lib/npm.js

```diff
@@ -92,7 +92,7 @@
   var commandCache = {}
   var aliasNames = Object.keys(aliases)
 
-  var littleGuys = [ 'isntall', 'verison' ]
+  var littleGuys = [ 'install', 'verison' ]
```

Comment:
you missed "verison" which is a misspelling of "version" - but these are also intentional.

these little guys? i wouldn't worry about these little guys.

Reply:
Ok. Thanks for the answer.